core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -539,6 +546,15 @@ class CityInfo {
         civInfo.updateDetailedCivResources() // this building could be a resource-requiring one
     }
 
+    fun getGoldForCapturingCity(conqueringCiv: CivilizationInfo): Int {
+        val baseGold = 20 + 10 * population.population + Random().nextInt(40)
+        val turnModifier = max(0, min(50, civInfo.gameInfo.turns - turnAcquired)) / 50f
```

Comment:
since civInfo.gameInfo.turns - turnAcquired is always >=0, then it or 50 /50 will always be >=0, 
so why do we need the max()?

Reply:
Better safe than sorry? :smile:  It's just my standard way of clipping an unknown value to a certain range. Of course as you noted, if everything works fine, `civInfo.gameInfo.turns - turnAcquired` is always >=0, so it isn't necessary.